funcx_endpoint/funcx_endpoint/endpoint/endpoint.py

```diff
@@ -90,6 +90,16 @@ def start_endpoint(
     endpoint_uuid : str
     """
     endpoint_dir = os.path.join(manager.funcx_dir, name)
+
+    if not os.path.exists(endpoint_dir):
+        msg = (f'\nEndpoint {name} is not configured!\n'
+               '1. Please create a configuration template with:\n'
+               f'\tfuncx-endpoint configure {name}\n'
+               '2. Update the configuration\n'
+               '3. Start the endpoint\n')
+        print(msg)
+        raise FileNotFoundError(f'Endpoint {name} is not configured!')
```

Comment:
I think it should probably keep the old style. No need to raise an exception here, and this exception seems irrelevant.

Reply:
Tried to raise a similar exception as
```
FileNotFoundError: [Errno 2] No such file or directory: '/home/zhuozhao/.funcx/test_local_1/config.py'
```
We can keep the old style.